client/src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,20 @@
+import Menu from "../Menu/Menu";
+import "./Navbar.css";
+
+export default function Navbar() {
+  const redirectionToHome = () => {
+    window.location.href = "/";
```

Comment:
Prefer using React router navigate function to avoid reloading your app here

Reply:
I added usenavigate